File: Ruby/TP8/Ej3/point.rb
class Point
  def initialize(x,y)
    @x = x
    @y = y
  end
  def to_s
    '{'+@x.to_s+','+@y.to_s+'}'
  end
  def get_x
    @x
  end
  def get_y
    @y
  end
  def distance(aux)
    aux_x= (aux.get_x-@x)**(2)
    aux_y = (aux.get_y-@y)**(2)
    Math.sqrt(aux_x+aux_y)

  end
end

File: Java/PrimerParcial/TP3/ej9/Point.java
package ar.itba.edu.POO.PrimerParcial.TP3.ej9;

public class Point {
    protected double x, y;

    public Point(double x, double y) {
        this.x = x;
        this.y = y;
    }

    public double getX() {
        return x;
    }

    public double getY() {
        return y;
    }
    public double distanceTo(Point point){
        return Math.sqrt(Math.pow(point.getX()-x,2)+(Math.pow(point.getY()-y,2)));
    }

    @Override
    public String toString() {
        return String.format("{%g, %g}", x, y);
    }
}